documentation/asciidoc/computers/raspberry-pi/introduction.adoc

```diff
@@ -27,12 +27,12 @@ a|
 512MB | 26-pin GPIO header
 a|
 * HDMI
-* 2 × USB 2.0
+* 2× USB 2.0
```

Comment:
Probably worth updating https://github.com/raspberrypi/style-guide/blob/master/style-guide.md#numerals-and-numbers to clarify what the spacing rules around the multiplication symbol are! (i.e. "2 × USB 2.0" versus "2× USB 2.0" versus "2 ×USB 2.0" versus "2×USB 2.0")